Documentation/GetPnPSubWebs.md

```diff
@@ -4,16 +4,18 @@ Returns the subwebs of the current web
 ```powershell
 Get-PnPSubWebs [-Recurse [<SwitchParameter>]]
                [-Web <WebPipeBind>]
+               [-Includes <String[]>]
                [-Identity <WebPipeBind>]
 ```
 
 
 ## Returns
->[List<Microsoft.SharePoint.Client.Web>](https://msdn.microsoft.com/en-us/library/microsoft.sharepoint.client.web.aspx)
+>[Microsoft.SharePoint.Client.WebCollection](https://msdn.microsoft.com/en-us/library/microsoft.sharepoint.client.web.aspx)
```

Comment:
I assume you made this change manually? This file is auto generated based upon the type specified in line 13 of the cmdlet class, that's why it said List<Microsoft.SharePoint.Client.Web> and not Microsoft.SharePoint.Client.WebCollection. Rebuilding the solution will overwrite this file again with the value specified. Besides that, WebCollection is a builtin class, whereas we actually return a generic list of type Web.

Reply:
Didn't manually modify the file. Probably changed it without this file being generated again. What do you want me to do so this PR can be submitted?

Reply:
@erwinvanhunen, any updates on this one?